<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Anvar&amp;Arthur Map</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="screen">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>700</width>
      <height>500</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="address_input">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>671</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="search">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>570</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Поиск</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="layout">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>10</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Схема</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="satellite">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>40</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Спутник</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="hybrid">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>70</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Гибрид</string>
    </property>
   </widget>
   <widget class="QLabel" name="address_output">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>671</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="reset_address">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>510</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Сброс</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="index_state">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>535</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>С индексом</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
